doctor_visits/delphi_doctor_visits/update_sensor.py

```diff
@@ -140,7 +147,7 @@ def update_sensor(
     params = Weekday.get_params(data) if weekday else None
 
     # handle explicitly if we need to use Jeffreys estimate for binomial proportions
-    jeffreys = True if se else False
+    jeffreys = se
```

Comment:
if this var needed?

Reply:
Yes, it's used in `sensor.py`

Reply:
oh i meant if it's going to be  jeffreys = se`, why not just use the `se` variable and not create/assign it to the `jeffreys` varrable?

Reply:
Yeah that's fair. I'll change it